File: Server/src/main/java/fr/intech/roomies/model/grocery/Item.java
package fr.intech.roomies.model.grocery;

import fr.intech.roomies.model.people.Roomie;

import javax.persistence.*;
import java.util.List;

@Entity
public class Item {

    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    private int itemId;

    @Column(nullable = false)
    private String name;

    @Column(nullable = false)
    private double price;

    private int quantity;
}

File: Roomies2.0/src/Roomies2.DAL/Model/Grocery/Item.cs
#region

using System;

#endregion

namespace Roomies2.DAL.Model.Grocery
{
    public class Item
    {
        public Item()
        {
            
        }

        public int ItemId { get; set; }
        public string ItemName { get; set; }
        public int UnitPrice { get; set; }
    }
}